### ISSUE TITLE
check_ping not working on Ubuntu 20.04 and Ubuntu 22.04

### ISSUE THREAD

**Issue by l3system** [sensu/monitoring-plugins#49]
When running the plugin I get this error:
CRITICAL - You need more args!!!
Could not open pipe:

Example:
./check_ping  -H 1.1.1.1 -w 10,10% -c 20,20%
CRITICAL - You need more args!!!
Could not open pipe:

All other plugins (check_disk, etc) work fine.
plugin works fine on CentOS 8.

Assets:
sensuctl asset list| grep ubuntu
sensu/monitoring-plugins  //assets.bonsai.sensu.io/.../monitoring-plugins-ubuntu1804_2.10.5_linux_amd64.tar.gz  cca3d6b  
sensu/monitoring-plugins  //assets.bonsai.sensu.io/.../monitoring-plugins-ubuntu2004_2.10.5_linux_amd64.tar.gz  f118bbd  
sensu/monitoring-plugins  //assets.bonsai.sensu.io/.../monitoring-plugins-ubuntu2204_2.10.5_linux_amd64.tar.gz  4ff9454

**Comment by jsadowyj** [comment]
Yep. This has been an issue forever, and nobody is maintaining it. We forked the repo and implemented a fix:
https://bonsai.sensu.io/assets/mnx-solutions/monitoring-plugins/versions/2.10.7

**Comment by l3system** [comment]
> Yep. This has been an issue forever, and nobody is maintaining it. We forked the repo and implemented a fix: https://bonsai.sensu.io/assets/mnx-solutions/monitoring-plugins/versions/2.10.7

Thanks a lot for the info!
